CONTRIBUTING.md

```diff
@@ -1,150 +0,0 @@
-# Welcome
```

Comment:
`CONTRIBUTING.md` is needed at root path for the project, we can quote it in other places if needed.
cc @kevin-wangzefeng

Reply:
Not sure the rule about this :(

Reply:
`CONTRIBUTING.md` is generally needed in the project.
If the content is duplicated in with others in /docs, you can simply add link to the files.

Reply:
Get it